lua/autorun/xadmin_init.lua

```diff
@@ -6,55 +6,95 @@ xAdmin = {
 	Groups = {},
 	Commands = {},
 	AdminChat = {},
+	Utility = {},
 	Github = true -- This can be used to identify the github xAdmin from the gmodstore xAdmin 
 }
 
-print("Loading xAdmin")
+local debugEnabled = false
+local function debugPrint(msg)
+	if not debugEnabled then return end
+	print("------------------------------------XADMIN DEBUG------------------------------------", msg)
+end
 
-xAdmin.Core.Version = "1.0"
+-- Load permissions before everything else to it doesnt break shit
+-- TOOD: Find a better, non shit way of doing this!
+if SERVER then
+	AddCSLuaFile("xadmin/_config/sh_permissions.lua")
+	include("xadmin/_config/sh_permissions.lua")
+else
+	include("xadmin/_config/sh_permissions.lua")
+end
 
-local path = "xadmin/"
+local intro = [[
+--------------------------------------------------------------
+xAdmin - R.I.P PoliceRP.xyz
+--------------------------------------------------------------
+Contributers to xAdmin in order of commits:
+https://github.com/owainjones74 | 63
+https://github.com/MilkGames | 12
+https://github.com/realpack | 3
+https://github.com/ExtReMLapin | 2
+https://github.com/DrPepperG | 2
+https://github.com/Livaco | 2
+https://github.com/LivacoNew | 1
+https://github.com/NoSharp | 1
+https://github.com/Roni-sl | 1
+--------------------------------------------------------------
+
+]]
+
+print(intro)
```

Comment:
If you're only setting the variable to immediately print it, maybe just pass the string itself into the print?